src/js/components/AccordionPanel/AccordionPanel.js

```diff
@@ -1,16 +1,24 @@
 import React, { Component } from 'react';
 import { compose } from 'recompose';
 
+import { withTheme } from 'styled-components';
+
+import { normalizeColor } from '../../utils';
+import { base } from '../../themes';
+
 import { Box } from '../Box';
 import { Button } from '../Button';
 import { Collapsible } from '../Collapsible';
 import { Heading } from '../Heading';
-import { withTheme, withForwardRef } from '../hocs';
-import { normalizeColor } from '../../utils';
+import { withForwardRef } from '../hocs';
 
 import { AccordionContext } from '../Accordion/AccordionContext';
 
 class AccordionPanel extends Component {
+  static defaultProps = {
```

Comment:
Instead of hardcoding the `theme: base` in every component, it might be better to create a base `defaultProps` object that gets mixed into each component, allowing you to import and set the default base theme in one place.

```js
static defaultProps = {
   ...defaultProps,
}
```

This will make it easier to refactor in the future.